ukasquared/README.md

```diff
@@ -0,0 +1,17 @@
+## Calculator
+This is a computational program which is built to assist in basic calculations of whole numbers
+
+## technologies used
+* C
+
+## Features of the calculator
++ Addition
++ Subtraction
++ Multiplication
++ Division
+
+## Compilation
+To use this calculation, follow the below steps:
+- [] using your terminal
+- [] run gcc main.c calc/_func.c
```

Comment:
I think you have a typo here. Your file is called `calc_func.c` not `calc\_func.c`

Reply:
Oh! That's right. Thanks